server/static/js/board.js

```diff
@@ -8,11 +8,11 @@ $('.grid').masonry({
 
 function popluateBoard(bobbles) {
   var bobbles_list = $("#bobbles");
```

Comment:
I think bobbleList would better fit standard JS style. Beyond that, variables that reference jQuery selections will typically have a $ in front so I think this could end up being `$bobbleList`